src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
+import java.util.Scanner;
```

Comment:
Hello! This class is generally well written! A few suggestions from me: Perhaps you want to call the hortizontalLine a variable so you can just call that instead of doing System.out.print(".....")!

You can consider doing the above^ as a final variable too since you will not be changing your horizontalLine